weave/decorator_op.py

```diff
@@ -21,6 +21,10 @@ def op(
     *,  # Marks the rest of the arguments as keyword-only.
     plugins=None,
     mutation=False,
+    # If True, the op will be weavified, ie it's resolver will be stored as a Weave
+    # op graph. The compile node_ops pass will expand the node to the weavified
+    # version, instead of executing the original python resolver body.
+    weavify=False,
```

Comment:
it seems like there are 2 kinds of "node ops" now:
1. Those that are data-independent (weavifiable)
2. Those that are data-dependent ("node_ops" - which require weave.use)